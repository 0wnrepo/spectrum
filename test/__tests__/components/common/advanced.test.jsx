import React from 'react';
import renderer from 'react-test-renderer';

import Advanced from '~/components/common/advanced';

it('renders with default title', () => {
  const tree = renderer.create(<Advanced>Test</Advanced>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders with custom title', () => {
  const tree = renderer.create(<Advanced title="test">Test</Advanced>).toJSON();
  expect(tree).toMatchSnapshot();
});
