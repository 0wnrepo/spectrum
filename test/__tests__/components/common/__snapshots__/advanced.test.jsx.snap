// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with custom title 1`] = `
<div
  className="field"
>
  <a
    onClick={[Function]}
    role="button"
    style={
      Object {
        "cursor": "pointer",
      }
    }
    tabIndex={0}
  >
    test
     
    <i
      aria-hidden="true"
      className="caret right icon"
    />
  </a>
</div>
`;

exports[`renders with default title 1`] = `
<div
  className="field"
>
  <a
    onClick={[Function]}
    role="button"
    style={
      Object {
        "cursor": "pointer",
      }
    }
    tabIndex={0}
  >
    Advanced
     
    <i
      aria-hidden="true"
      className="caret right icon"
    />
  </a>
</div>
`;
